src/JuliusSweetland.OptiKey.Core/Models/XmlKeyboardModels/XmlDynamicKey.cs

```diff
@@ -31,6 +31,9 @@ public string ReturnToThisKeyboardAsString
             set { this.ReturnToThisKeyboard = XmlUtils.ConvertToBoolean(value); }
         }
 
+        public string Scratchpad
```

Comment:
What does string Scratchpad on an XmlDynamicKey do?

Reply:
That's a mistake. It should be removed.

Reply:
Could you please replace this PR with this change made? Thx